src/domain/entities/figma-design-identity.ts

```diff
@@ -0,0 +1,59 @@
+const FIGMA_DOCUMENT_NODE_ID = '0:0';
+
+export class FigmaDesignIdentity {
```

Comment:
nit: sorry @akostevich-atlassian I've only just had a chance to look at this. Thoughts on renaming to `FigmaDesignIdentitfier`?

Reply:
No worries, Jon: your review is welcome anytime! :)

We could definitely explore other options! I called it `FigmaDesignIdentitfier` since it conceptually represents a _compound ID_ of a Figma design (`fileKey` + `nodeId`) -- it is a quite standard name for this kind of entities.

Do you have, Jon, any options in mind, so I could understand better in which direction you are thinking?

Reply:
Sorry might be getting terms confused, in code it's currently defined as `FigmaDesignIdentity`, I was thinking we could change it to `FigmaDesignIdentifier`. Which one do you prefer?

Reply:
Ah, I see. Yes, I agree, `FigmaDesignIdentifier` will be a more precise option. Let me address this in a separate PR later today.